controllers/utils/policy_util.go

```diff
@@ -226,7 +248,62 @@ func UpdateManagedPolicyNamespaceList(policyNs map[string][]string, policyNameAr
 			break
 		}
 	}
-	if nsFound == false {
+	if !nsFound {
 		policyNs[crtPolicyName] = append(policyNs[crtPolicyName], crtPolicyNs)
 	}
 }
+
+// StripObjectTemplatesRaw removes all the ACM raw templating from a string and returns an interface
+// of what the object-templates would be if not for the raw templating
+func StripObjectTemplatesRaw(tmplStr string) string {
+
+	// Create a copy of the input since we will be editing it multiple times
+	result := tmplStr
+
+	// Get all variable usages
+	variableUsageRegex := regexp.MustCompile(`{{\s*\$[\w\.]*\s*}}`)
+	variableUsages := variableUsageRegex.FindAllStringSubmatch(result, -1)
+
+	// Get the usage of all of those variables
+	for _, item := range variableUsages {
+		// item[0] is the full usage of a variable
+		// e.g. "{{ $my.example.var }}"
+
+		// Replace all plain usages of variable with placeholder
+		result = strings.ReplaceAll(result, item[0], "placeholder")
```

Comment:
Maybe make `placeholder` a constant in utils.constants since it's used quite a lot.